tagmanager/ctags/ctags.c

```diff
@@ -49,6 +49,7 @@
 
 
 
+#ifndef GLIB_MAJOR_VERSION
```

Comment:
I was about to delete the entire block, but ctags.c also contains checks for AMIGA, __EMX__, MS-DOS, QDOS and whatnot, so how much deletion would be enough?.. Since there are several #if 0 blocks, I decided to put this one in an #ifndef which will both exclude it from compilation and signify why.

It doesn't really matter if the block will be deleted (though there are others which deserve deletion much more), or excluded with #if 0, or something else, so please say what you consider the right thing. After daf4dd45b874f7d29e30f8eddd4fbb6cae40e687 was applied, this PR is conflicting, so I'll have to modify or recreate it anyway.